src/psbt.h

```diff
@@ -54,7 +54,7 @@ static constexpr uint8_t PSBT_SEPARATOR = 0x00;
 
 // BIP 174 does not specify a maximum file size, but we set a limit anyway
 // to prevent reading a stream indefinitely and running out of memory.
-const std::streamsize MAX_FILE_SIZE_PSBT = 100000000; // 100 MiB
+const std::streamsize MAX_FILE_SIZE_PSBT = 100000000; // 100 MB
```

Comment:
These kind of comments are really asking to go out of sync, would be fine with removing it as well.
Would using  C++14 digit separators be an idea maybe? `100'000'000`?

Reply:
My preference would be to use constants like [this one](https://github.com/bitcoin/bitcoin/pull/25386) to make the code equivalent to this documentation:  `100 * MB_BYTES`.